crates/llms/src/openai/mod.rs

```diff
@@ -27,8 +31,11 @@ use async_openai::{
 use async_trait::async_trait;
 use futures::future::try_join_all;
 use serde_json::Value;
+use server::Server;
 use snafu::ResultExt;
 
+pub mod server;
+
 const MAX_COMPLETION_TOKENS: u16 = 1024_u16; // Avoid accidentally using infinite tokens. Should think about this more.
 
 pub(crate) const GPT3_5_TURBO_INSTRUCT: &str = "gpt-3.5-turbo";
```

Comment:
Let's also add GPT-4o.

Reply:
we'll be able to specify which to use in the model config.